qml/Launcher/Drawer.qml

```diff
@@ -69,7 +69,8 @@ FocusScope {
 
     Rectangle {
         anchors.fill: parent
-        color: "#BF000000"
+        color: theme.palette.normal.background
+        opacity: 0.99
```

Comment:
"BF" is 0.6875, not 0.99. Seems like opacity being 0.99 might as well be 1.0 at that point. Probably 0.6875 is a bit too transparent though, so maybe something closer to 0.90 would be best? Maybe something between 0.85 and 0.93?

Reply:
I tried that, but the transparency was still really distracting when the background was white (so, every app in Ubuntu Touch right now).